index.js

```diff
@@ -1256,6 +1256,47 @@ Victor.prototype.isEqualTo = function(vec2) {
 	return this.x === vec2.x && this.y === vec2.y;
 };
 
+/**
+ * Returns a true if this vector is parallel to another
+ *
+ * ### Examples:
+ *     var vec1 = new Victor(20, 40);
+ *     var vec2 = new Victor(60, 120);
+ *     vec1.isParallelTo(vec2);
+ *
+ *     // => true
+ *
+ * @return {Boolean}
+ * @api public
+ */
+ Victor.prototype.isParallelTo = function(vec2) {
+    if (vec2.x === 0) return this.x === 0;
+    if (vec2.y === 0) return this.y === 0;
+
+    var kx = this.x / vec2.x;
+    var ky = this.y / vec2.y;
+    
+    if (kx !== ky || ky !== 0) return false;
+    else return true;
```

Comment:
the current code will always return false unless vec2.y === Infinity

the better way to do this and avoid division:

if you put the vectors as columns of a matrix, the columns are parallel if the matrix is linearly dependent, so we can just check if the determinant of the matrix is 0. this gives x1 * y2 - x2 * y1 = 0 which can be rewritten as x1 * y2 = x2 * y1. this also covers the edge case you have at the top because multiplying by 0 will just give 0.

```suggestion
    return this.x * vec2.y === this.y * vec2.x;
```